### ISSUE TITLE
Impossible to download old version of ComfyUI

### ISSUE THREAD

**Issue by DiegoRRR** [comfyanonymous/ComfyUI#4286]
### Expected Behavior

Downloading a file

### Actual Behavior

Download fails every time after about 460MB. I tried in several different browsers.

### Steps to Reproduce

- Go on https://github.com/comfyanonymous/ComfyUI/releases
- Try to download https://github.com/comfyanonymous/ComfyUI/releases/download/latest/ComfyUI_windows_portable_nvidia_cu118_or_cpu_28_04_2023.7z
(This particular version is important to Windows 7 users)

### Debug Logs

```powershell
Download failed, needs authorization.
```

**Comment by ltdrdata** [comment]
Check if you have enough free space on your disk.

**Comment by DiegoRRR** [comment]
It's not a space problem, I downloaded other bigger files later and also tried on another computer. There is a problem on the website.

**Comment by rhutkovich** [comment]
I've downloaded via wget & Google Chrome without any issues. Probably something with your internet connection.

**Comment by DiegoRRR** [comment]
I just tried again, in three different browsers and on two different computers, still always fails after 400MB.
I live in France, I thought maybe an european CDN used by github has a broken copy of this file, so I tried downloading using a proxy in USA. I tried several times, it always eventually fails after 1.1GB now.
Using a proxy in Europe it always fails after 400MB, exactly like when trying to download directly.

**Comment by ltdrdata** [comment]
> I just tried again, in three different browsers and on two different computers, still always fails after 400MB. I live in France, I thought maybe the european CDN used by github has a broken copy of this file, so I tried downloading using a proxy in USA. I tried several times, it always eventually fails after 1.1GB now. Using a proxy in Europe it always fails after 400MB, exactly like when trying to download directly.

Have you tried downloading using a VPN? If the same issue still occurs, you might need to suspect that something within your local machine is blocking it.

**Comment by DiegoRRR** [comment]
I don't know if it's a coincidence but I tried this good old trick of downloading adding "?d=1" at the end of the url to force it to give me a fresh original copy of the file instead of some cache or CDN, and it worked at last. Then I tried again with just the usual url and it still works now. This really seems like a broken copy of the file was cached somewhere. Or maybe it's just a coincidence. I guess I can close the issue.
